pkg/cmd/run.go

```diff
@@ -82,13 +83,6 @@ func initializeRunOpts(o *globals.GlobalOpts, reference string) error {
 		}
 		runOpts.WorkingDir = workingDir
 	}
-	if runOpts.Log == nil { // not overridden for tests
-		runOpts.Log = log.New(os.Stdout, "run: ", log.LstdFlags)
-	}
-	if runOpts.DebugLog == nil { // not overridden for tests
-		// All debug features are optional. If there is any unexpected failure, log as "debug" to stdout.
-		runOpts.DebugLog = log.New(os.Stdout, "debug: ", log.LstdFlags)
-	}
```

Comment:
looks like `func Run(o *globals.GlobalOpts, cmd *cobra.Command, args []string) error {` below is unused now..

Reply:
great find. I really wish goland was as good as you at finding unused exported stuff :D